opennebula/resource_opennebula_virtual_machine.go

```diff
@@ -314,6 +315,41 @@ func resourceOpennebulaVirtualMachineRead(d *schema.ResourceData, meta interface
 		return err
 	}
 	return nil
+
```

Comment:
this is dead code due to the return nil above

Reply:
you are totally right